nibabel/streamlines/compact_list.py

```diff
@@ -0,0 +1,232 @@
+import numpy as np
+
+
+class CompactList(object):
+    """ Class for compacting list of ndarrays with matching shape except for
```

Comment:
So there is no maximum line length for the first line of the doctring? Or, I should try to make it smaller?